### PR TITLE
fades 5 (new formula)

### PR DESCRIPTION
**Pull request by filipeximenes**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

_You can erase any parts of this template not applicable to your Pull Request._

### New Formulae Submissions:

- [x] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

### CODE REVIEW

File: Library/Formula/fades.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    pyver = Language::Python.major_minor_version "python3"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python#{pyver}/site-packages"
+    system "python3", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    system "#{bin}/fades", "--version"
```
Comment (dunn) [inline Library/Formula/fades.rb]: Is a more substantial test possible; something that checks a bit more of the functionality?  [`hugo.rb`](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/hugo.rb) and [`cmake.rb`](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/cmake.rb) are good examples.

**Comment by filipeximenes** [comment]
@dunn thanks for the references!
Updated.

**Comment by filipeximenes** [comment]
any updates here?

**Comment by dunn** [comment]
Looks good! Will merge when green.

**Comment by dunn** [comment]
@BrewTestBot test this please

**Comment by apjanke** [comment]
Merged. Thank you for your contribution to Homebrew!

**Pull request closed without merging**